### PR TITLE
(DOCUMENT-1001) Clarify installation dependencies

### PR DESCRIPTION
**Pull request by eimlav**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ VMware vSphere is a cloud computing virtualization platform. The vSphere module
 
 ### Installing the vSphere module
 
+The following are *dependencies* of the module and as such should be installed on the system from which the module is configured to be used on i.e. in a master-agent setup, the *agent* is the one that requires these dependencies installed.
```
Comment (clairecadman) [inline README.md]: Looks good, but maybe reword along the lines of:
"The following are *dependencies* of the module. Install these on the system which you configure the module on. For example, in a master-agent setup, install the dependencies on the agent".

Comment (eimlav) [reply README.md]: Updated.

**Comment by clairecadman** [comment]
@eimlav shall I go ahead and merge?

**Comment by eimlav** [comment]
@clairecadman Yea that would be great Claire cheers!
